src/dsp_tools/commands/excel2json/models/new_lists_deserialise.py

```diff
@@ -5,12 +5,6 @@
 import pandas as pd
 
 
-@dataclass
-class ExcelFile:
-    filename: str
-    sheets: list[ExcelSheet]
-
-
 @dataclass
```

Comment:
In this file there is now only this class. But I want to keep that for the future. Currently checks are happening in the serialisation class. I want to change that so that it is more similar to the json header. So I think it is ok to keep this file as it is.